src/averageAges.js

```diff
@@ -37,7 +47,28 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const arrayWomen = people.filter(item => item.sex === 'f');
+  let arrayMothers = [];
+
+  if (withChildren) {
+    for (let i = 0; i < arrayWomen.length; i++) {
+      for (let j = 0; j < people.length; j++) {
+        if (arrayWomen[i].name === people[j].mother) {
+          arrayMothers.push(arrayWomen[i]);
+        }
+      }
+    }
+  } else {
+    arrayMothers = arrayWomen;
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/70009205/209686312-a4bd83d1-7ff2-4648-a194-28a482ad5084.png)
fix it everywhere